liquibase-dist/src/main/maven/release.pom.xml

```diff
@@ -39,7 +39,7 @@
         <dependency>
             <groupId>org.yaml</groupId>
             <artifactId>snakeyaml</artifactId>
-            <version>1.33</version>
+            <version>2.0</version>
```

Comment:
It would be better if we had a property in the topmost parent pom for the snakeyaml version, so we only had to change this in one place in the future.

Reply:
That's true, this way I would not have missed this one. Maybe this should be adressed when we restructure/rename the projects?